test cases/common/178 bothlibraries/meson.build

```diff
@@ -54,7 +54,8 @@ test('runtest-both-2', exe_both2)
 # both_libraries the static has no sources and thus no compilers, resulting in
 # the executable linking using the C compiler.
 # https://github.com/Netflix/vmaf/issues/1107
-libccpp = both_libraries('ccpp', 'foo.cpp', 'libfile.c')
+libccpp = both_libraries('ccpp', 'foo.cpp', 'libfile.c',
+  cpp_args : ['-std=c++11'])
```

Comment:
Hmm, this seems wrong TBH. We should be using default_options to set cpp_std for this, probably? Or per-target override_options. I wonder what this actually does on MSVC. :/